src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -711,22 +712,43 @@ export class ChatWidget extends Disposable implements IChatWidget {
 	}
 
 	private renderWelcomeViewContentIfNeeded() {
+
+		// reset the input in welcome view if it was rendered in experimental mode
+		if (this.container.classList.contains('experimental-welcome-view')) {
```

Comment:
nit and not blocking, but can maybe do a `classlist.toggle` on the startupExpValues here instead of checking + `add/remove` here and at line 740